doc/ref_model/chapters/chapter03.rst

```diff
@@ -1178,7 +1178,7 @@ The considerations will help to guide the build and deployment of the Storage so
 |    | can bring with it considerations that are specific to this choice             |
 +----+----+----+----------+----------------------------------------------------------+
 |    | Converged Software Defined Storage                                            |
-|----+----+----+----------+----------------------------------------------------------+
++----+----+----+----------+----------------------------------------------------------+
```

Comment:
```suggestion
+---------+----+---------------------------------------------------------------------+
```

Reply:
It is difficult to edit multiline stuff in the GitHub UI for comments, so I'm, not sure if this is totally correct, but I hope you get the idea.